example/life_game/README.md

```diff
@@ -0,0 +1,37 @@
+# Conway's Game of Life
+
+This example is a [Conway's Game of Life](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life) implementation.
+3 outputs are proposed (console, [OCaml Graphics](https://github.com/ocaml/graphics) and HTML/JavaScript),
+each one using the same WebAssembly modules.
+
+## Console et OCaml Graphics
+
+Console and Graphics implementations use *external functions* technique as the
+[standard example](../). Here are the command lines to run in this directory to execute the programs:
+
+* Console  mode: `dune exec -- ./life_graphics.exe`
+* Graphics mode: `dune exec -- ./life_console.exe`
+
+Several modules (defined in `life.wast` and `life_loop.wast` files) are linked.\
+Why 2 .wast files are required ?\
+Because, in JavaScript, the `Sleep` function uses the
+[Promise system](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise)
+which cannot work in a wasm call context.
+So, if we want to use the same WebAssembly components in all contexts,
+the game loop had to be extracted in a separate module.\
+See the following section for JavaScript implementation details.
+
+## HTML/JavaScript
+
+Here is a standard example of a wasm module called in a web page using the
+[JavaScript API](https://webassembly.org/getting-started/js-api/).
+
+First of all, a compiled version of the WebAssembly module is needed, obtained by the
+command line `wat2wasm life.wast`. wat2wasm is part of the
+[WABT: The WebAssembly Binary Toolkit](https://github.com/WebAssembly/wabt).
+
+As in OCaml in the previous contexts, external functions are defined in the environment execution
+(here in JavaScript). As explained before, only one wasm module is imported (`life.wasm`) and
+the game loop has been specifically written.
```

Comment:
I have trouble understanding the end of this sentence maybe you meant the following:

> and the game loop has been written separately

`separately` as in, in a separate module ?